var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -42,7 +45,7 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
     git = "https://github.com/cp2k/cp2k.git"
     list_url = "https://github.com/cp2k/cp2k/releases"
 
-    maintainers("dev-zero", "mtaillefumier")
+    maintainers("dev-zero", "mtaillefumier", "msimberg", "RMeli")
```

Comment:
@msimberg?

Reply:
Happy to be added as maintainer here. I completely missed this PR, so it would be good to be automatically notified.

Reply:
Thanks for the ping @RMeli! Yeah, I'm fine with this as well.